src/compiler/transformers/component-native/native-static-style.ts

```diff
@@ -33,20 +33,20 @@ const addMultipleModeStyleGetter = (
       const styleLiteral = createStyleLiteral(cmp, style);
```

Comment:
GH won't allow me to comment on a line higher than this one - can we add a comment before this conditional starts (above the `if` statement) that the order of the conditional must match `add-static-style.ts#getSingleStyle`?

Reply:
on this point, I was thinking it might make sense to circle back after the bugfix is out and factor out this logic into a helper function since it looks essentially the same in these two spots